frontend/components/LiveQuery/SelectTargets.tsx

```diff
@@ -325,7 +325,12 @@ const SelectTargets = ({
     if (isFetchingCounts) {
       return (
         <>
-          <Spinner small />
+          <Spinner
```

Comment:
Can we add more padding to the left of this so it aligns with the text it replaces?


https://user-images.githubusercontent.com/71795832/194648769-7097f0d9-00ae-4237-9107-b11f43c2db6b.mov

Reply:
done